package.json

```diff
@@ -6,5 +6,9 @@
   ],
   "devDependencies": {
     "lerna": "^5.3.0"
+  },
+  "dependencies": {
```

Comment:
this is the wrong place for these dependencies, you want to update packages/greenbox-script/package.json